template/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like we move the template from the root of the repo. Can we remove the template from the repo root?

Reply:
Where would you like these stored? Happy to toss them wherever. @iVegas

Reply:
Sorry, nevermind. I just realized that we moved into the template folder recently.